gradle.properties

```diff
@@ -1,8 +1,9 @@
 version=1.0
 dockerTag=devel
-artifactory_contextUrl=https://gtie-artifactory.us.dell.com/artifactory
 dependency_repo=libs-snapshot
 buildInfo.licenseControl.runChecks=true
 buildInfo.licenseControl.autoDiscover=true
 buildInfo.build.name=com.dell.isg.smi:service-device-discovery
-systemProp.sonar.host.url=http://100.68.126.201:9000
\ No newline at end of file
+bintrayRepo=smi-service-device-discovery
+bintrayUserOrg=rackhd-smi
+bintrayVcsUrl=https://rackhd-smi.bintray.com/smi-service-device-discovery
```

Comment:
change to "https://bintray.com/rackhd/docs/apidoc"

Reply:
done